### ISSUE TITLE
[Feature Request] Separate camera/hand bobbing, sprint indication

### ISSUE THREAD

**Issue by ghost** [FlashyReese/sodium-extra-fabric#84]
Hi. First off, I appreciate your work on this mod and it's very nice to use.
In Minecraft you can disable dynamic FOV which is nice for people who get motion sick like me. A consequence of this is that it's very hard to know when you're sprinting and when you aren't. 
I came to the idea of combining a feature in Sk1er's Patcher mod in which camera movement/bobbing is disabled, but the hand bobbing is kept, **but in this implementation only when sprinting!**
In effect, static FOV, no hand bobbing when walking/idling, but when you sprint your hand bobs. 
This also would add the feature of separate toggles for camera & hand bobbing, but also with a toggle for "only when sprinting".
tl;dr separate options to disable camera & hand/viewmodel bobbing, option under to enable viewmodel bobbing only while sprinting which serves as a sprint indication.
